codegenerator/cli/npm/envio/index.d.ts

```diff
@@ -1 +1,110 @@
-export { logger as Logger } from "./src/Envio.gen";
+export type {
+  logger as Logger,
+  effect as Effect,
+  effectContext as EffectContext,
+  effectArgs as EffectArgs,
+  effectOptions as EffectOptoins,
+} from "./src/Envio.gen.ts";
```

Comment:
This is hand written right?

Do we have tests that ensure it stays correct and compiles if things change?